e2e/utilities/utils.ts

```diff
@@ -32,7 +32,13 @@ export function getKAIPluginName(): string {
 export async function cleanupRepo() {
   if (fs.existsSync(repoDir)) {
     try {
-      await execPromise(`rm -rf "${repoDir}"`);
+      if (getOSInfo() == 'windows') {
+        await execPromise(
+          `powershell.exe Remove-Item -Recurse -Force "${repoDir}"`
+        );
+      } else {
+        await execPromise(`rm -rf "${repoDir}"`);
+      }
```

Comment:
Is there a reason why we're using OS-specifif commands instead of using the `fs` package of node? 

`fs.rmSync(dir, { recursive: true, force: true });`

Reply:
no I just followed the pattern fo the existing code and though to add os check because the rm doesn't work on windows. I'll change it.

Reply:
Fixed